### PR TITLE
Feature/pcrr 143 invest

### PR DESCRIPTION
**Pull request by stompudown**

### CODE REVIEW

File: src/components/organisms/invest/helper/set-field-values.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const setFieldValues = (setFieldValue, values, depositAmount) => {
```
Comment (ketsuo) [inline src/components/organisms/invest/helper/set-field-values.tsx:1]: Please rename the method. e.g calculateFieldValues

File: src/config/network.ts [hunk 2]
```diff
@@ -1,17 +1,20 @@
-import { ChainId } from '@usedapp/core'
+import {ChainId} from '@usedapp/core'
 
 export const supportedChains = [
   ChainId.Mainnet,
   ChainId.Polygon,
   ChainId.Mumbai,
+    ChainId.Kovan
```
Comment (ketsuo) [inline src/config/network.ts:7]: Please remove kovan. We dont use in this stage.

File: src/config/network.ts [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
 ]
 
 export const supportedStakingChains = [
   ChainId.Polygon,
   ChainId.Mumbai,
+  ChainId.Kovan
```
Comment (ketsuo) [inline src/config/network.ts:13]: Please remove kovan. We dont use in this stage.

File: src/config/network.ts [hunk 4]
```diff
@@ -1,17 +1,20 @@
...
 ]
 
 export const supportedVestingChains = [
   ChainId.Polygon,
   ChainId.Mumbai,
+  ChainId.Kovan
```
Comment (ketsuo) [inline src/config/network.ts:19]: Please remove kovan. We dont use in this stage.

File: src/providers/strategies.ts [hunk 5]
```diff
@@ -26,7 +26,8 @@ export const strategyProvider: IStrategyProvider = {
     },
     chainAddresses: {
       [ChainId.Mainnet]: '09309932',
-      [ChainId.Polygon]: '09309932'
+      [ChainId.Polygon]: '0xAaD7328bCC81AC7D8ae9187d7F10B503E3d38Eb8',
+      [ChainId.Kovan]: '0xAaD7328bCC81AC7D8ae9187d7F10B503E3d38Eb8'
```
Comment (ketsuo) [inline src/providers/strategies.ts:30]: Please remove kovan. We dont use in this stage and all from below code.

**Review by ketsuo — CHANGES_REQUESTED** [review]

File: src/components/organisms/invest/helper/set-field-values.tsx (hunk 1 above)
Comment (stompudown) [reply src/components/organisms/invest/helper/set-field-values.tsx]: Done

File: src/providers/strategies.ts (hunk 5 above)
Comment (stompudown) [reply src/providers/strategies.ts]: Done

File: src/config/network.ts (hunk 4 above)
Comment (stompudown) [reply src/config/network.ts]: Done

File: src/config/network.ts (hunk 3 above)
Comment (stompudown) [reply src/config/network.ts]: Done

File: src/config/network.ts (hunk 2 above)
Comment (stompudown) [reply src/config/network.ts]: Done

**Review by ketsuo — APPROVED** [review]

**Pull request merged**